pkg/logger/s3_log_writer.go

```diff
@@ -36,13 +40,38 @@ func CreateBucket(minioClient *minio.Client, bucketName string) error {
 }
 
 func (l *S3LogWriter) putLog(msg string) {
+	now := time.Now()
+
+	level := "info"
+
+	if len(msg) > 0 {
+		if strings.HasPrefix(msg, "✗ ") {
```

Comment:
```suggestion
		if strings.HasPrefix(msg, "✗") {
```
without the space would be just fine :-)

Reply:
Removed the space :)

Reply:
And in the other place too.